pkg/webui/console/components/mac-settings-section/index.js

```diff
@@ -119,56 +184,130 @@ const MacSettingsSection = props => {
         <Radio label={sharedMessages['16Bit']} value={FRAME_WIDTH_COUNT.SUPPORTS_16_BIT} />
         <Radio label={sharedMessages['32Bit']} value={FRAME_WIDTH_COUNT.SUPPORTS_32_BIT} />
       </Form.Field>
-      {isABP && (
+      {(isABP || isOTAA) && (
         <>
+          <Form.FieldContainer horizontal>
+            {!isOTAA && (
+              <Form.Field
+                title={m.rx1DelayTitle}
+                type="number"
+                tooltipId={tooltipIds.RX1_DELAY}
+                name="mac_settings.rx1_delay"
+                component={Input}
+                min={1}
+                max={15}
+                inputWidth="xxs"
+              />
+            )}
+            <Form.Field
+              title={m.desiredRx1DelayTitle}
+              type="number"
+              name="mac_settings.desired_rx1_delay"
```

Comment:
```suggestion
              tooltipId={tooltipIds.RX1_DELAY}
              name="mac_settings.desired_rx1_delay"
```

We can use the same tooltip.